contexts/arbo-context/filter-update-steps/apply-new-selected-filters.tsx

```diff
@@ -14,10 +14,12 @@ export function filterData(
   return data.filter((item: any) => {
     return filterKeys.every((key: string) => {
       /* If no pathogen is selected, we don't want to see any data */
-      if (key === "pathogen"){
-        if(filters[key].length == 0){
-          return false;
-        }
+      if (key === "pathogen" && filters[key].length == 0 ){
+        return false;
+      }
+      /* Ignore pediatric age group for non pediatric age groups. */
+      if (key === "pediatricAgeGroup" && item["ageGroup"] !== "Children and Youth (0-17 years)"){
+        return true;
```

Comment:
This behaviour is inconsistent with the serotype filter. Are you sure this is what we want?

Reply:
I agree, it is inconsistent. But when we asked about how they wanted the serotype filter to behave, they said they wanted it to behave differently.